charts/rancher-turtles/questions.yml

```
@@ -1,47 +1,60 @@
 namespace: rancher-turtles-system
 questions:
 - variable: rancherTurtles.features.default
-  default: "true"
-  description: "Enable extra features"
-  label: Enable extra features
+  default: "false"
+  description: "Customize install setting"
+  label: Customize install setting
   type: boolean
-  show_subquestion_if: false
+  show_subquestion_if: true
   group: "Rancher Turtles Extra Features"
```

Maybe something like this:

```suggestion
  group: "Rancher Turtles Custom Settings"
```